src/Symfony/Component/Intl/Tests/NumberFormatter/NumberFormatterTest.php

```diff
@@ -50,7 +50,7 @@ public function testConstructorWithPatternDifferentThanNull()
     public function testSetAttributeWithUnsupportedAttribute()
     {
         $formatter = $this->getNumberFormatter('en', NumberFormatter::DECIMAL);
-        $formatter->setAttribute(NumberFormatter::LENIENT_PARSE, null);
+        $formatter->setAttribute(NumberFormatter::LENIENT_PARSE, 100);
```

Comment:
what would it take to *not* change the test case? (same below)

Reply:
this is linked to https://github.com/symfony/symfony/pull/32525#discussion_r303137603

the second parameter of `setAttribute` is an int according to https://php.net/manual/fr/numberformatter.setattribute.php

Test is trying to pass an unsupported attribute (`NumberFormatter::LENIENT_PARSE` in this case) and assert that `MethodArgumentValueNotImplementedException` is thrown. Since the `null` value now throws a `TypeError`, test fails. So I've put a dummy int just keep the execution ongoing and reach the expected exception.